packages/slate/src/models/history.js

```diff
@@ -53,6 +53,27 @@ class History extends Record(DEFAULTS) {
     )
   }
 
+  /**
+   * Create a list of `Operations` from `operations`.
+   *
+   * @param {Array<Object>|List<Object>} operations
+   * @return {List<Object>}
+   */
+
+  static createList(operations = []) {
```

Comment:
Hey @t3chguy can you rename this to `createOperationsList`?

Reply:
Sure